tests/testthat/test-busy-indication.R

```diff
@@ -0,0 +1,40 @@
+test_that("useBusyIndicators()", {
+  res <- tagList(
+    useBusyIndicators(),
+    useBusyIndicators(spinners = FALSE),
+    useBusyIndicators(pulse = FALSE),
+    useBusyIndicators(spinners = FALSE, pulse = FALSE),
+  )
+  expect_snapshot(res)
```

Comment:
```suggestion
  expect_snapshot(
    tagList(
      useBusyIndicators(),
      useBusyIndicators(spinners = FALSE),
      useBusyIndicators(pulse = FALSE),      
      useBusyIndicators(spinners = FALSE, pulse = FALSE),
    )
  )
```

The snapshot is pretty confusing otherwise